corehq/apps/reports/util.py

```diff
@@ -88,6 +88,13 @@ def get_all_users_by_domain(domain=None, group=None, user_ids=None,
         Returns a list of CommCare Users based on domain, group, and user 
         filter (demo_user, admin, registered, unknown)
     """
+    def _create_temp_user(user_id):
+        username = get_username_from_forms(domain, user_id).lower()
```

Comment:
that shouldn't mattter. this is only for users that aren't `CommCareUser`s so this is only used for a user that has submitted a form that isn't a registered user